com.yacreader.YACReader.yml

```diff
@@ -0,0 +1,123 @@
+app-id: com.yacreader.YACReader
+runtime: org.kde.Platform
+runtime-version: "5.15-22.08"
+sdk: org.kde.Sdk
+command: YACReader
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --filesystem=home
```

Comment:
can't the filesystem permissions be narrowed?

Reply:
Library management needs directory level access to scan files and create a database. Portals can't provide that and if we limit access to something more restricted like Documents and Downloads only this will be experienced as an unnecessary restriction by the users.
Even worse, during my tests Qt indicated a "driver not loaded" error when the QtSql driver could not access the directories, so lots of bug reports would be expected.